runtime-extension-model/src/main/resources/META-INF/mule-operation.xsd

```diff
@@ -39,6 +39,13 @@
                             </xsd:documentation>
                         </xsd:annotation>
                     </xsd:element>
+                    <xsd:element name="deprecated" type="deprecatedType" minOccurs="0">
```

Comment:
Shouldn't this have `maxOccurs="1"`? And I think the same would go for the deprecation construct inside operations, right?

Which also makes me wonder... how is all of this working without having to modify `MuleOperationExtensionModelDeclarer`? I mean, the operations have the [deprecation construct as a NestedComponent](https://github.com/mulesoft/mule/blob/c588c59ace9408923aa66193d6b1af5a130f1a2f/runtime-extension-model/src/main/java/org/mule/runtime/core/api/extension/MuleOperationExtensionModelDeclarer.java#L163), but the parameters don't.

Reply:
maxOccurs=1 -> yeah, I just copied from other place, I'll fix it

declarer thing -> THANK YOU, I completely forgot it